deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/resource/Groups.java

```diff
@@ -212,4 +217,18 @@ public void actionOnHealthCheckState(@Context SecurityContext sc,
         configHistoryHandler.updateConfigHistory(groupName, Constants.TYPE_ASG_GENERAL, groupBean, operator);
         configHistoryHandler.updateChangeFeed(Constants.CONFIG_TYPE_GROUP, groupName, Constants.TYPE_ASG_GENERAL, operator);
     }
+
+    @GET
+    @Path("/{groupName: [a-zA-Z0-9\\-_]+}/policy")
+    public Map<String, ScalingPolicyBean> getGroupPolicy(@Context SecurityContext sc,
+                                                         @PathParam("groupName") String groupName) throws Exception {
+        return awsAutoScalingManager.getScalingPoliciesForGroup(groupName);
+    }
+
+    @GET
+    @Path("/{groupName: [a-zA-Z0-9\\-_]+}/capacity/desired")
```

Comment:
Instead of creating a new API, can we reuse the existing one?

@GET
    @Path("/summary")
    public AutoScalingSummaryBean getAutoScalingSummaryByName(@PathParam("groupName") String groupName) throws Exception {
        return groupHandler.getAutoScalingSummaryByName(groupName);
    }